governance/governance.go

```diff
@@ -44,6 +44,7 @@ const CONTAINER_GOVERNOR = "ContainerGovernor"
 const MICROSERVICE_GOVERNOR = "MicroserviceGovernor"
 const BC_GOVERNOR = "BlockchainGovernor"
 const SERVICE_CONFIGSTATE_GOVERNOR = "ServiceConfigStateGovernor"
```

Comment:
I think this constant is not used any more

Reply:
Done